mhkit/river/performance.py

```diff
@@ -104,7 +104,7 @@ def tip_speed_ratio(rotor_speed,rotor_diameter,inflow_speed):
     '''
     Function used to calculate the tip speed ratio (TSR) of a MEC device with rotor
 
-    Parameters
+    Parameters:
```

Comment:
This is a weird one related to my initial concerns when i posted this PR. I know this change had already been implemented in a previous PR months ago (as with a lot of the commits that are labeled as "ahead" of the master). Rebecca had mentioned it might be the case where I worked on something before doing a git pull and this caused the commit history to get screwy. Anyways, I will update this particular change. My main concern is that when this gets merged into the master, it doesn't mess up its commit history somehow. I am not much of an expert on Github so maybe it's harmless, but just wanted to throw that out there.